body.go

```diff
@@ -1,12 +1,32 @@
 package postman
 
+//These constants represent the available raw languages.
```

Comment:
Please add a space after `//` :
```suggestion
// These constants represent the available raw languages.
```

Reply:
done